source/views/home/home.js

```diff
@@ -16,6 +16,7 @@ import {allViews} from '../views'
 import {HomeScreenButton, CELL_MARGIN} from './button'
 import {trackedOpenUrl} from '../components/open-url'
 import {EditHomeButton} from '../components/nav-buttons'
+import {OpenSettingsButton} from '../components/nav-buttons'
```

Comment:
Please stick this import inside the previous import, so that we don't import the same file twice.